docs/releases/1.1.txt

```diff
@@ -288,7 +288,7 @@ A couple of small -- but highly useful -- improvements have been made to the
 test client:
 
 * The test :class:`Client` now can automatically follow redirects with the
-  ``follow`` argument to :meth:`Client.get` and :meth:`Client.post`. This
+  ``follow`` parameter to :meth:`Client.get` and :meth:`Client.post`. This
```

Comment:
Refers to parameter below:
https://github.com/lachilles/django/blob/3db84b5526a16a136862381c6f8b08c59da4cb82/django/test/client.py#L514-L514